src/late_multicellular_stage/MulticellularHUD.tscn

```diff
@@ -152,6 +170,7 @@ offset_right = 150.0
 offset_bottom = 25.0
 fit_content = true
 ExtendedBbcode = "HOLD_TO_SHOW_CURSOR_ADVICE_TEXT"
+EnableTooltipsForMetaTags = null
```

Comment:
I think you have some godot broken accidental commits of stuff later in this file which all should be removed...